packages/eas-cli/src/devices/actions/create/urlMethod.ts

```diff
@@ -0,0 +1,32 @@
+import chalk from 'chalk';
+import indentString from 'indent-string';
+import qrcodeTerminal from 'qrcode-terminal';
+import { URL } from 'url';
+
+import { getExpoWebsiteBaseUrl } from '../../../api';
+import { createAppleDeviceRegistrationRequestAsync } from '../../../credentials/ios/api/AppleDeviceRegistrationRequest';
+import { AppleTeam } from '../../../credentials/ios/api/AppleTeam';
+import log from '../../../log';
+
+export async function runUrlMethodAsync(accountId: string, appleTeam: AppleTeam): Promise<void> {
```

Comment:
```suggestion
export async function displayDeviceRegisterInfo(accountId: string, appleTeam: AppleTeam): Promise<void> {
```
not sure what url method means